wear/build.gradle

```diff
@@ -97,14 +99,16 @@ dependencies {
 
     implementation project(':modules:services:analytics')
     implementation project(':modules:services:compose')
+    implementation project(':modules:services:images')
     implementation project(':modules:services:localization')
+    implementation project(':modules:services:model')
     implementation project(':modules:services:preferences')
+    implementation project(':modules:services:repositories')
+    implementation project(':modules:services:servers')
```

Comment:
Adding the servers module as a dependency is the only substantive change here. The rest are just recording the declarations.